activities/member-relations/collect-information.md

```diff
@@ -0,0 +1,32 @@
+---
+type: Resource
+---
+
+# What information to collect about members for member relations 
+
+## Introduction
+
+This resource outlines the information that Foundation for Public Code staff use to map member relations.
+
+The information collected will be used to structure and keep track of our ongoing relationships with members. It is based on our [mission](../../organization/mission.md) and the first iterations of the [founding member proposal template](founding-membership-proposal.md). In line with our communication principles, this should be concise and hypertextual where possible. 
+
+As information collected may include personal or sensitive information, this is not public but stored in a private and secure environment.
+
+## Information collected 
+
+* Background: ```Lists existing work and/or projects relevant to the Foundation's mission.``` 
+   * Aligned projects: ```Provide a short description of notable public-purpose open and collaborative software and policy projects```
+   * Opportunity projects: ```Provide a short description of notable software and policy projects with strong potential to become more open, collaborative and/or public-purpose```
+   * Additional initiatives: ```Provide a short description of other initiatives not directly related to, but adjacent to our mission```
+
+* Relationship to the Foundation for Public Code
+   * Decision makers: ```Outline the top two or three budget holders, policy makers and managers who decide on Foundation membership```
+   * Internal champions: ```Outline the top two or three internal champions who can help promote our mission and act as ambassadors internally```
+   * Internal skeptics: ```Outline the top two or three internal stakeholders who are skeptical of our mission, for whom we can develop and test our value proposition mapping and assumptions```
+* Concrete value propositions and next steps
+   * Title 1: ```Include a title that is actionable and descriptive, such as "assess for stewardship", and reference a project in 'Background'```
```

Comment:
The hierarchy no longer makes sense to me here - there's a disconnect between the heading 'Concrete value propositions and next steps' and 'title' underneath it. Is the intention for the items at the 'title' level to be:
* problems we can solve for the member (which is how we've used 'value mapping propositions' elsewhere)
* potential activities or programmes of work to undertake with the member

It sounds like the titles are potential programmes of work, in which case we might need to hone our vocabulary to distinguish a member's existing projects vs our programmes of work with them.

NB: @bvhme considers the word 'project' to be paradigmatically wrong for software powering services, which is only over when the service is replaced or decomissioned.

Reply:
Good points. made some edits, but feel the language is still a bit muddly. As these are quite important terms in our vocabulary I suggest we have a short meeting to define what we think they should be

Reply:
Let's make the 'project' discussion an issue as it related to the whole repo